health-services/libraries/health-services-common/src/main/java/org/egov/common/data/repository/GenericRepository.java

```diff
@@ -85,7 +85,9 @@ public List<T> findById(List<String> ids, Boolean includeDeleted) {
         paramMap.put("ids", ids);
 
         objFound.addAll(namedParameterJdbcTemplate.query(query, paramMap, rowMapper));
-        putInCache(objFound);
+        if(objFound.size() > 0 ){
```

Comment:
can we move this block to putInCache? so that it fixes the problem for any method that wants to use putInCache?

Reply:
done